javascript/code-challenges/challenges-11.test.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function named transformToLis that, given an object, returns an array of the key value pairs as html list items.
+
+For example:
+{
+  name: 'bob',
+  age: 32
+}
+
+Becomes:
+[
+<li>name: bob</li>,
+<li>age: 32</li>
+]
+------------------------------------------------------------------------------------------------ */
+
+function transformToLis(obj){
+  // Solution code here...
+  for(const [key, value] of Object.entries(obj)){
+    document.createElement("li")
+    li.innerHTML = `${key}: ${value}`
+    document.ol.appendChild(li);
+    //let list = `${key}: ${value}`
+    // return list
+    // console.log(list)
+  }
+
+  
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named count that, given an integer and an array of arrays, uses either filter, map, or reduce to count the amount of times the integer is present in the array of arrays.
+
+Note: You might need to use the same method more than once.
+
+For example, count(5, [[1, 3, 5, 7, 9], [5, 5, 5], [1, 2, 3]]) returns 4.
+------------------------------------------------------------------------------------------------ */
+
+const count = (target, input) => {
+  // let count = 0;
+  // for(let i = 0; i < input.length; i++){
+  //   for(let j = 0; j < column.length; j++){
+  //     let number = column.filter(x => x = target)
+      
+  //     newArr.push(number)
+  //   }
+  // }
+  // return newArr.length;
+  // let newArr = []
+  // input.forEach(arr => {
+  //   arr.forEach((number) =>{
+  //     let targetNumber = arr.filter(x => x = input);
+  //     //console.log(targetNumber)
+  //   })
+  // });
+  // return newArr;
```

Comment:
Nice work so far on challenges!

Feedback: Remove this code if it is not in use to keep code readable and clean 👍